src/qibolab/_core/instruments/qm/config/__init__.py

```diff
@@ -1,3 +1,5 @@
-from .config import Configuration
-from .devices import ControllerId, ModuleTypes
-from .pulses import SAMPLING_RATE, operation
+from .config import Configuration as Configuration
+from .devices import ControllerId as ControllerId
+from .devices import ModuleTypes as ModuleTypes
+from .pulses import SAMPLING_RATE as SAMPLING_RATE
+from .pulses import operation as operation
```

Comment:
What was wrong with the original code? Why do we need to redefine the imported name if we are not planning to change it?

I see a similar change in a couple of other files.

Reply:
Indeed, this is something I have done manually.

The reason is that this is an arising pattern (maybe *arisen*, more than *arising*...), which is aiming to distinguish unused imports from explicit re-exports.

I was already aware of it, but I didn't bother until now. Since it was suggested by the linter (assessing at the same time the whole amount of fixes required), I took the chance to implement it